src/File/SimpleRelativePathHelper.php

```diff
@@ -8,15 +8,19 @@ class SimpleRelativePathHelper implements RelativePathHelper
 	/** @var string */
 	private $currentWorkingDirectory;
 
-	public function __construct(string $currentWorkingDirectory)
+	public function __construct(string $currentWorkingDirectory, string $directorySeparator = DIRECTORY_SEPARATOR)
 	{
-		$this->currentWorkingDirectory = $currentWorkingDirectory;
+		$this->currentWorkingDirectory = rtrim($currentWorkingDirectory, $directorySeparator);
+		if ($currentWorkingDirectory === '') {
+			return;
+		}
+		$this->currentWorkingDirectory .= $directorySeparator;
```

Comment:
Hi, there's a convention in PHPStan's codebase that all directories are without the trailing directory separator. It follows how `__DIR__` or `dirname()` works. Can you rewrite the logic so that it's also true here? Thanks.